scst-store/multicluster-setup.hbs.md

```diff
@@ -17,25 +17,32 @@ cluster to communicate with SCST - Store in the View cluster.
 > profile](#install-build-profile) step, instead of deploying the Build profile
 > again, update your deployment using `tanzu package installed update`.
 
+![Screenshot of the SCST - Store in Multi Cluster Deployment](images/multicluster-deployment.jpg)
+
 ## Prerequisites
 
 You must install the View profile. See [Install View profile](../multicluster/installing-multicluster.hbs.md#install-view).
 
 ## Procedure summary
 
-1. Copy SCST - Store CA certificate from the View cluster.
+1. Copy CA certificates from the View cluster.
+   1. Copy Metadata Store CA certificate from the View cluster.
+   2. Copy AMR CloudEvent Handler CA certificate from the View cluster.
 2. Copy SCST - Store authentication token from the View cluster.
-3. Apply the CA certificate and authentication token to the Kubernetes cluster where you intend to install the Build profile.
+   1. Copy Metadata Store authentication token from the View cluster.
+   2. Copy AMR CloudEvent Handler edit token from the View cluster.
+3. Apply the Metadata Store CA certificates, Metadata Store authentication token and CloudEvent Handler edit token to the Kubernetes cluster where you intend to install the Build profile.
 4. Install the Build profile.
 
 ## Copy SCST - Store CA certificate from View cluster
 
-With your kubectl targeted at the View cluster, you can view SCST - Store's TLS
+### Copy Metadata Store CA certificate from the View cluster
```

Comment:
⛏️Update to: `Copy CA certificate from the View cluster`?  It would align it with the Procedure summary item.

Reply:
Discussed and resolving